src/Bellatrix.Core/settings/ConfigurationService.cs

```diff
@@ -49,9 +49,10 @@ private static IConfigurationRoot InitializeConfiguration()
         var builder = new ConfigurationBuilder();
         var executionDir = ExecutionDirectoryResolver.GetDriverExecutablePath();
         var filesInExecutionDir = Directory.GetFiles(executionDir);
+        var configuration = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyConfigurationAttribute>().Configuration;
         var settingsFile =
 #pragma warning disable CA1310 // Specify StringComparison for correctness
-                filesInExecutionDir.FirstOrDefault(x => x.Contains("testFrameworkSettings") && x.EndsWith(".json"));
+                filesInExecutionDir.FirstOrDefault(x => x.Contains("testFrameworkSettings") && x.Contains(configuration) && x.EndsWith(".json"));
```

Comment:
if I remember correctly we resolved this without this change?

Reply:
Yep, we resolved it.